steps/metadata.js

```diff
@@ -35,6 +35,7 @@ module.exports = async function getMetadata(argv, cli) {
   cli.write(metadata);
   cli.separator();
 
+  data.warnCommits = argv.warnCommits;
   const checker = new PRChecker(cli, data);
```

Comment:
A better way to do this is just passing `argv` to the `PRChecker` constructor instead of passing them/monkey-patching them here and there...

Reply:
@joyeecheung done and added a test since, now can pass a flag.